CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Sweep AI
+
+First off, thank you for considering contributing to Sweep AI. It's people like you that make Sweep AI such a great tool.
+
+## Creating Issues
+
+Before you submit an issue, please do a search in [open issues](https://github.com/sweepai/sweep/issues) to see if the issue or feature request has already been filed.
+
+Use the provided issue template when creating a new issue. Fill in the template with as much detail as possible. The more detail you provide, the more likely that someone can help you.
```

Comment:
Add that Sweep can also create tickets. Try getting Sweep to create the ticket first before manually trying it.

Reply:
[]()

Reply:
> []()